cfg/config.go

```diff
@@ -12,19 +12,20 @@ import (
 	"strings"
 )
 
-// Allowed logger levels & config key
+// Allowed logger levels & config key.
 const (
 	DebugLogLvl     = "DEBUG"
 	InfoLogLvl      = "INFO"
 	ErrorLogLvl     = "ERROR"
 	LogLvlConfigKey = "LOG_LEVEL"
+	lenOfLines      = 2
 )
 
-// Load configs from a env file & sets them in environment variables
+// Load configs from a env file & sets them in environment variables.
 func loadEnvVar() error {
 	f, err := os.Open(".env")
 	if err != nil {
-		return err
+		return fmt.Errorf("error while opening .env file: %w", err)
```

Comment:
If you change the name of the file in the code above - you most probably will forget to change it in this wrap message. Let's better use a constant and wrap the value into this error.

Reply:
fix